src/main/java/task7/tests/cloudTests/CreatingFolderTest.java

```diff
@@ -0,0 +1,50 @@
+package task7.tests.cloudTests;
+
+import org.testng.Assert;
+import org.testng.annotations.*;
+import task7.Entities.Browser;
+import task7.bo.*;
+import task7.screens.MailRuCloudMainPage;
+import task7.services.LoginService;
+
+public class CreatingFolderTest {
+    private Folder folder = FolderFactory.getFolderWithUniqueName();
+    private MailRuCloudMainPage cloudPage = new MailRuCloudMainPage();
+
+    @BeforeClass
+    public void setUp() {
+        User user = UserFactory.getUserWithValidCredentials();
+        LoginService.LoginToMailRuCloud(user);
+    }
+
+    @BeforeMethod
+    public void cleanBeforeTest() {
+        cloudPage.switchToTheTabByIndex(2)
+                .clickSelectAllButton()
+                .cleanCloudBeforeTest();
+    }
```

Comment:
For classes with one test method, it would be better to combine all these flows in BeforeMethod to avoid possible issues and conflicts.